### PR TITLE
Sync develop changes July 2 - July 19 to hdf5_1_14

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

**Review by byrnHDF — APPROVED** [review]

File: HDF5Examples/C/H5G/test-pc.sh [hunk 1]
```diff
@@ -0,0 +1,235 @@
+#! /bin/sh
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```
Comment (jhendersonHDF) [inline HDF5Examples/C/H5G/test-pc.sh:4]: Probably needs removed as this is a new file and doesn't appear in some of the other new .sh files.

File: HDF5Examples/C/H5T/test-pc.sh [hunk 2]
```diff
@@ -0,0 +1,250 @@
+#! /bin/sh
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```
Comment (jhendersonHDF) [inline HDF5Examples/C/H5T/test-pc.sh:4]: Probably needs removed as this is a new file and doesn't appear in some of the other new .sh files.

File: HDF5Examples/FORTRAN/H5G/test-pc.sh [hunk 3]
```diff
@@ -0,0 +1,151 @@
+#! /bin/sh
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```
Comment (jhendersonHDF) [inline HDF5Examples/FORTRAN/H5G/test-pc.sh:4]: Probably needs removed as this is a new file and doesn't appear in some of the other new .sh files.

File: HDF5Examples/FORTRAN/H5T/test-pc.sh [hunk 4]
```diff
@@ -0,0 +1,264 @@
+#! /bin/sh
+#
+# Copyright by The HDF Group.
+# Copyright by the Board of Trustees of the University of Illinois.
```
Comment (jhendersonHDF) [inline HDF5Examples/FORTRAN/H5T/test-pc.sh:4]: Probably needs removed as this is a new file and doesn't appear in some of the other new .sh files.

File: doxygen/aliases [hunk 5]
```diff
@@ -3,18 +3,27 @@ ALIASES += THG="The HDF Group"
...
 # doxygen subdir (develop, v1_14)
 ALIASES += DOXURL="hdfgroup.github.io/hdf5/v1_14"
 #branch name (develop, hdf5_1_14)
 ALIASES += SRCURL="github.com/HDFGroup/hdf5/blob/hdf5_1_14"
-
+#Other projects that contribute to HDF5
+ALIASES += PRJURL="\HDFURL/projects"
+ALIASES += HVURL="github.com/HDFGroup/hdfview/blob/master"
+ALIASES += PLURL="github.com/HDFGroup/hdf5_plugins/blob/master"
+>>>>>>> e716059... Update doxygen Learn Basics / example refs. Add Reference sections (#4640)
```
Comment (jhendersonHDF) [inline doxygen/aliases:26]: git diff line was accidentally added here

**Pull request merged**